modepy/quadrature/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""
+.. currentmodule:: modepy
+
+.. autoclass:: Quadrature
+
```

Comment:
```suggestion
.. autoclass:: TensorProductQuadrature
.. autoclass:: GaussLegendreTensorProductQuadrature
```
?

Should also generalize `TensorProductQuadrature` to non-homogeneous nD, to match `TensorProductBasis`.